src/Miunie.Discord/Entities/HelpResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿// This file is part of Miunie.
+//
+//  Miunie is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  Miunie is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with Miunie. If not, see <https://www.gnu.org/licenses/>.
+
+using System.Collections.Generic;
+
+namespace Miunie.Discord.Entities
+{
+    public class HelpResult
+    {
+        public string Title { get; set; }
+
+        public List<HelpSection> Sections { get; } = new List<HelpSection>();
```

Comment:
Instead of using a concrete `List`, we should use one of the abstract variants, for example `IEnumerable` or `ICollection` if `Add` is important.